admin-dev/themes/new-theme/js/pages/feature-flag/index.js

```diff
@@ -31,9 +31,24 @@ $(() => {
   const $submitButton = $('#submit-btn-feature-flag');
   const $form = $('#feature-flag-form');
 
+  let noFieldModifiedInTheForm = true;
+
+  $form.find('input').change(() => {
+    noFieldModifiedInTheForm = false;
```

Comment:
What about saving the formData before any action is made so you can compare it to the formData when submitting and see if the form has been really modified or not? (Because now if you toggle twice the same switch, you'll get the alert when you probably should not as no data has been changed)

Reply:
That is a good idea 👍

Reply:
Done, and PR rebased